net/cifs-utils/Makefile

```diff
@@ -8,23 +8,25 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=cifs-utils
-PKG_VERSION:=6.4
-PKG_RELEASE:=2
+PKG_VERSION:=6.7
+PKG_RELEASE:=1
 
 PKG_SOURCE_URL:=https://download.samba.org/pub/linux-cifs/cifs-utils/
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
-PKG_HASH:=38fc63926af435dae4ebcf4406275580a692d9fb9ee3e32170317cf2ba68e6e3
+PKG_HASH:=b2f21612474ab012e75accd167aab607a0614ff67efb56ea0f36789fa785cfab
 
 PKG_MAINTAINER:=Florian Fainelli <florian@openwrt.org>
 PKG_LICENSE:=GPL-3.0
 PKG_LICENSE_FILES:=COPYING
 
+PKG_FIXUP:=libtool
+
 include $(INCLUDE_DIR)/package.mk
 
 define Package/cifsmount
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+kmod-fs-cifs
+  DEPENDS:=+kmod-fs-cifs +libtalloc
```

Comment:
Was this always a dependency, but it's only being called out now?  Or do newer versions introduce this dependency?

Reply:
Can you try building it without this?  What happens if you do?

Reply:
Okay, asked and answered... it fails.